corehq/apps/domain/templates/domain/stripe_cards.html

```diff
@@ -0,0 +1,93 @@
+{% load hq_shared_tags %}
+{% load i18n %}
+
+<fieldset>
+    <legend>{% trans 'Saved Credit Cards' %}</legend>
+    <table class='table'>
+        <thead>
+            <tr>
+                <th style="width:20%;"></th>
+                <th style="width:20%;"></th>
+                <th style="width:20%;"></th>
+                <th style="width:20%;"></th>
+                <th style="width:20%;"></th>
+            </tr>
+        </thead>
+        <tbody>
+            <!-- ko foreach: cards -->
+                <tr>
+                    <td>
+                        <i class="icon-credit-card"></i> <strong><span data-bind="text: brand"></span></strong>
+                    </td>
+                    <td>
+                        ************<strong><span data-bind="text: last4"></span></strong>
+                    </td>
+                    <td><span data-bind="text: exp_month"></span>/<span data-bind="text: exp_year"></span></td>
+                    <td>
+                        <span class="label label-success"
+                              data-bind="visible: is_autopay">
+                            {% trans 'Autopay card' %}
+                        </span>
+                        <button data-bind="visible: is_autopay,
+                                           click: unSetAutopay,
+                                           enable: $parent.autoPayButtonEnabled"
+                                class='btn btn-danger autopay-button'>
+                            {% trans 'Remove' %}
+                        </button>
+                        <button data-bind="visible: !is_autopay(),
+                                           click: setAutopay,
+                                           enable: $parent.autoPayButtonEnabled"
+                                class='btn autopay-button'>
+                            {% trans 'Set as autopay card' %}
+                        </button>
+                    </td>
+                    <td>
+                        <button data-bind="click: toggleDeleteConfirmation,
+                                           visible: !showDeleteConfirmation()"
+                                class="btn btn-danger">
+                            <i class="icon-remove"></i> {% trans 'Delete Card' %}
+                        </button>
+                        <div class="alert"
+                             style="margin-bottom:0;"
```

Comment:
no, not that i know of. don't think adding a style in a css sheet really makes it better. it's just not always great to modify the bootstrap classes. can't actually see what this looks like but sounds like it's the only option